src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_Regex.vb

```diff
@@ -271,6 +271,30 @@ class c
             End Using
         End Function
 
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestNegativeCategory(showCompletionInArgumentLists As Boolean) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                <Document><![CDATA[
+using System.Text.RegularExpressions;
+class c
+{
+    void goo()
+    {
+        var r = new Regex(@"\P$$");
```

Comment:
worth a test with target-typed expression?

Reply:
The code change doesn't involve that scenario, so I don't think that's worthwhile to add a part of this :-)